src/main/java/blossom/task/Deadline.java

```diff
@@ -0,0 +1,19 @@
+package blossom.task;
+
+public class Deadline extends Task {
+    protected String by;
```

Comment:
Maybe using a more descriptive name such as deadline or dueTime/dueDate might be better here.